pkg/execution/state/redis_state/queue.go

```diff
@@ -1573,6 +1573,16 @@ func (q *queue) partitionPeek(ctx context.Context, partitionKey string, sequenti
 			return nil, fmt.Errorf("error reading partition item: %w", err)
 		}
 
+		// NOTE: The queue does two conflicting things:  we peek ahead of now() to fetch partitions
+		// shortly available, and we also requeue partitions if there are concurrency conflicts.
+		//
+		// We want to ignore any partitions requeued because of conflicts, as this will cause needless
+		// churn every peek MS.
+		if item.ForceAtMS > ms {
+			ignored++
```

Comment:
could we instrument this value as well?

Reply:
We could, but I don't think it'll give us too much valuable information given it's mixed with allow/denylisting.